pyproject.toml

```diff
@@ -30,7 +30,7 @@ ignore_missing_imports = true
 
 
 [tool.pylint]
-master.py-version = "3.7"
+master.py-version = "3.8"
```

Comment:
ruff has a Python version too, right? otherwise UP036 would complain above

Reply:
Yeah, it does: `target-version = "py37"`

Reply:
This will go away if we move to scikit-build-core after 0.10 comes out. ;)

Reply:
Thanks for catching this @Skylion007!

Fixed with commit 49580cfa977eaef89b95d070e59214ebc9a05c43

That made me realize we need to add something like this to the list of useful `git grep` commands, for the next time when we want to drop a Python version:

```
git grep -I '[^0-9]37[^0-9]'
git grep -I '[^0-9]38[^0-9]'
```

Reply:
It will also go away if define the minimum python version in pyproject.toml standard location instead of setup.cfg :)